src/report_reader.cpp

```diff
@@ -349,22 +349,22 @@ DataFrame<T> ReportReader<T>::Population::get(const nonstd::optional<Selection>&
         node_ids = selection->flatten();
     }
 
+    std::vector<std::pair<uint64_t, uint64_t>> positions;
+    uint64_t min = UINT64_MAX;
+    uint64_t max = 0;
```

Comment:
The (nice) commit msg and presentation mentions why this min/max calc is happening, but perhaps add a small comment?

Reply:
Done! Thanks for the suggestion.